news/_posts/2024-12-19-community_page.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "GTN's Gift for 2024"
+contributions:
+  authorship:
+  - nomadscientist
+  - hexylena
+tags: [gtn, single-cell, new feature, new tutorial]
```

Comment:
```suggestion
tags: [gtn, single-cell, new feature, new tutorial, contributing, community]
```

(then this post will also show up on the 2024 in review page of the community and contributing topics)

Reply:
damnit, I literally wrote down 'make sure to add tags'. I forgot I'm that community too. Which I started...